docs/ref/templates/language.txt

```diff
@@ -74,6 +74,8 @@ explained later in this document.
 
     Oh, and one more thing: making humans edit XML is sadistic!
 
+.. _template-variables::
```

Comment:
The errors in the docs build are because you need a single colon here instead of double.